src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/01-create-books-table.yaml
+  - include:
+      file: db/changelog/changes/02-create-users-table.yaml
+  - include:
+      file: db/changelog/changes/03-create-roles-table.yaml
+  - include:
+      file: db/changelog/changes/04-create-users-roles-table.yaml
+  - include:
+      file: db/changelog/changes/05-add-admin-in-users.yaml
+  - include:
+      file: db/changelog/changes/06-insert-roles.yaml
+  - include:
+      file: db/changelog/changes/07-assign-roles.yaml
+  - include:
+      file: db/changelog/changes/08-insert-books.yaml
+  - include:
+      file: db/changelog/changes/09-set-bcrypt-passwords.yaml
```

Comment:
Please check all migration files and add empty line at the end of file where needed. This icon means that there is no empty line:
![image](https://github.com/ValeraLemberov/Book-Store-Implementation/assets/82539553/78e05117-fd60-4cd3-ab73-caec89997431)

Reply:
@vovasalyha  Thank you, I'll fix everything.